compiler/rustc_session/src/config/cfg.rs

```diff
@@ -262,6 +274,9 @@ impl CheckCfg {
 
         ins!(sym::debug_assertions, no_values);
 
+        ins!(sym::debug_fmt_detail, empty_values)
```

Comment:
Missing the entry in the [check-cfg - unstable book](https://doc.rust-lang.org/unstable-book/compiler-flags/check-cfg.html#well-known-names-and-values): `src/doc/unstable-book/src/compiler-flags/check-cfg.md`

*(don't forget to also update the date)*